applications/tari_merge_mining_proxy/src/proxy.rs

```diff
@@ -615,13 +619,27 @@ impl InnerService {
         }
 
         trace!(target: LOG_TARGET, "Sending request {:?}", request.body(),);
-        let resp = builder
+
+        let json_response;
+        if submit_block && !self.config.proxy_submit_to_origin {
+            // Assume it would be accepted
+            let id = json["id"].as_i64().unwrap_or_else(|| -1);
+            let accept_response = format!(
```

Comment:
Rather use the json macro 

```suggestion
            let accept_response =  json::json! {
               id: id,
               jsonrpc: "2.0",
               result: {},
               status: "OK",
               untrusted: false
            };
```

Reply:
Thanks, done.